src/library_pthread.js

```diff
@@ -262,6 +341,7 @@ var LibraryPThread = {
     };
     Atomics.store(HEAPU32, (pthread.threadInfoStruct + {{{ C_STRUCTS.pthread.threadStatus }}} ) >> 2, 0); // threadStatus <- 0, meaning not yet exited.
     Atomics.store(HEAPU32, (pthread.threadInfoStruct + {{{ C_STRUCTS.pthread.threadExitCode }}} ) >> 2, 0); // threadExitCode <- 0.
+    Atomics.store(HEAPU32, (pthread.threadInfoStruct + {{{ C_STRUCTS.pthread.profilerBlock }}} ) >> 2, 0); // profilerBlock <- 0.
```

Comment:
That's true. I wanted to keep this unconditional because to avoid having to rebuild two versions of libc, the struct layout is kept the same and so this field unconditionally exists, so I wanted to also unconditionally zero-initialize it to avoid any possibility of harm coming up in the form of a potentially dangling but unused pointer.